src/components/common/Map.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+import { bool, object } from 'prop-types';
+import { geolocated } from 'react-geolocated';
+
+import marker from '../../img/marker.svg';
+
+const Map = withScriptjs(withGoogleMap((props) => {
+  const { latitude, longitude } = props.coords;
```

Comment:
you could do _const { coords: { latitude, longitude }, isMarkerShown } = props;_ so you avoid doing _props.isMarkerShown_